concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1750,10 +1789,10 @@ jobs:
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: ['performance-tests']
+            passed: ['performance-tests', 'smoke-tests', 'acceptance-tests', 'custom-acceptance-tests', 'bosh-tests']
             trigger: true
```

Comment:
this adds a lot of lines to concourse. I am not sure this is necessary, as previous job of performance tests depends on all these, so they have to be passed if perf-test is passed....